validator/testdata/amp4email_feature_tests/css_length.out

```diff
@@ -22,6 +22,8 @@ PASS
 |  -->
 |  <!doctype html>
 |  <html ⚡4email>
+>> ^~~~~~~~~
```

Comment:
Is this an incomplete merge conflict?

Reply:
No, this is the AMP validator error format. See e.g. https://github.com/ampproject/amphtml/blob/005a72a03a56d2c23e2fd3a0fa8209a004f94bea/validator/testdata/amp4email_feature_tests/nonce_disallowed.out#L28-L29